### DISCUSSION TITLE
Implement a "quality control" there is too much garbage models there

### DISCUSSION THREAD

**Discussion by ExperimentDiffusion** [civitai/civitai#238] [Ideas]

**Comment by WASasquatch** [comment]
Kinda subjective when it comes to art, don't you think? Do you have examples of models? I have seen one or two "spam" models that are just other models trolling.

**Reply by ExperimentDiffusion** [reply]
Not really. For this "subjectiveness" there are art curators, museums, cultural managers, artists and art critics, books on the philosophy of art, and above all the time that puts everything in its place, the history of art. It is not subjective, human beings tend to disdain complexity and reduce the steps to get from one point to another, subjectivity is too complex, that's why artistic movements and labels emerge.

**Reply by ExperimentDiffusion** [reply]
Any quality model needs to be "trained" with a big bunch of examples and their relative "stems" the original paper instructs to train more than 400 examples of one "style" to produce the greatest variety of relations with other stems. Out there are models trained with 8 pictures and two words. Art is the art of be selective too, by selecting something you don't have to give examples of what is what. As an example I personally "like" SD1.5, MidJourney, Dreamshaper, Microworlds, Papercut, Counterfeit, Lineart, AnythingV3, just to mention some.

**Reply by WASasquatch** [reply]
> Not really. For this "subjectiveness" there are art curators, museums, cultural managers, artists and art critics, books on the philosophy of art, and above all the time that puts everything in its place, the history of art. It is not subjective, human beings tend to disdain complexity and reduce the steps to get from one point to another, subjectivity is too complex, that's why artistic movements and labels emerge.

This isn't an art museum. It's a community. Please provide examples. Seems right now you're just complaining and giving long winded replies, but fall short of an example of this. You say "too much" but do not provide a single example.

**Reply by candre23** [reply]
> Kinda subjective when it comes to art, don't you think? Do you have examples of models? I have seen one or two "spam" models that are just other models trolling.

It is subjective, but stuff like this is pretty clearly trash.  7GB checkpoint with no description and one really janky sample image: https://civitai.com/models/7586/kaya

There are a couple "groups" posting several new models per week that are all pretty low effort and have little in the way of description beyond "This doesn't work great but you should test it! Join our discord and support us on patreon!".  They may well be operating in good faith, but "DJZ" and "duskfall crew" have hundreds of space-wasting checkpoints between them. 

Personally, I'd suggest that all full checkpoints trained for a single face are unnecessarily wasteful at this point.  You want a face, train a TI or LORA and distribute that.  Nobody need another 2-8GB model just for one particular celebrity.

**Reply by poisenbery** [reply]
I don't want to list any names or call anyone out, but I've noticed a few models that are very clearly overcooked. "First" seems to be valued more than "good quality" in a lot of cases when it comes to models based around creating specific characters.
I don't really thing this is a subjective matter. If a model is unusable at weight 1, and the creator acknowledges that the model is overfit, it shouldn't be uploaded.

**Comment by EldarFara** [comment]
Can you point on some of garbage models?

**Reply by TheBarret** [reply]
See this is where the proposal falls apart, because nobody wants to be that person that turns down and then has to explain why he or she thinks this is not enough or enough to be up there. You cannot just go "It sucks" you need to have some good points before one makes such decision. So i like the idea, but the execution has to be sound else you will get drama and people will leave.
So for this to work, the owners and moderators of Civit need to come up with a solid hierarchy ranking system that is fair and also shows the progression for uploaders, the difference between an Advanced and a Novice user must be a journey where one learns and advances his status as well, that is chosen by the community.

**Comment by Raivshard** [comment]
I think this could be mitigated quite a bit by implementing more stringent posting protocols. Require a list of source models for merges. Require a stated purpose and style of the model. Require a version number. Require at least four different images generated by the model.

**Reply by candre23** [reply]
These suggestions would really go a long way to solving the trash problem.  There aren't _that_ many new models posted daily (yet) that they couldn't be actively moderated.  Even if it's just for first-time posters.  Your first upload gets sent to purgatory until a human looks over the post and decides whether it's good enough to go live or gets sent back to be filled out properly.

**Reply by WASasquatch** [reply]
> These suggestions would really go a long way to solving the trash problem. There aren't _that_ many new models posted daily (yet) that they couldn't be actively moderated. Even if it's just for first-time posters. Your first upload gets sent to purgatory until a human looks over the post and decides whether it's good enough to go live or gets sent back to be filled out properly.

Verified posting would go a long way in general. Your first entry into a category gets reviewed to see if it's for one, not just broken, and two, at least usable for it's intent.

**Comment by JohnMadden123456** [comment]
+100
Even if we talking about quality models, even those need moderators, or rather curators attention
E.g.: https://civitai.com/models/15283/rezero-garfiel
Looks fine, but what about tags? 

> Tags ANIME CHARACTER LORA

What is this trash? how about removing `lora` since it's already in the type, how about adding `man`, `re:zero`, `re:zero garfiel` and maybe couple others more? I'm not even touching chinese/japanese models/loras which usually don't even have english description and title, not only useful tags

**Reply by WASasquatch** [reply]
Yeah, no I think that's fine the way it is, and maybe you don't understand the system. The name is a tag as well, so `Re:Zero Garfiel
` is covered. The tags are fine, using lora, just like embedding, and textual inversion helps when you're searching without filters, etc. Nothing wrong here. Not trash at all, just a reason why this is subjective.

**Reply by JohnMadden123456** [reply]
If name is a tag, it should appear here https://civitai.com/tag/re%3Azero , but it doesn't and you need to do second separate search for name. Genres here are not defined yet like in books https://www.goodreads.com/genres/science-fiction or anime https://myanimelist.net/anime/genre/38/Military , but it doesn't mean collections/tags can't be organized based on all of those together - danboru tags, anime names, people names, photography styles, drawing techniques, etc.

**Reply by WASasquatch** [reply]
> If name is a tag, it should appear here https://civitai.com/tag/re%3Azero , but it doesn't and you need to do second separate search for name. Genres here are not defined yet like in books https://www.goodreads.com/genres/science-fiction or anime https://myanimelist.net/anime/genre/38/Military , but it doesn't mean collections/tags can't be organized based on all of those together - danboru tags, anime names, people names, photography styles, drawing techniques, etc.

My point is, it is the LORA's name, thus, searchable, indexable, etc.  And I don't think people need to be too excessive with their tagging and pollute searches too, and should be pretty narrow and explicit to the subject matter. Adding Re:Zero would clearly help *if you're for some reason relying on tags only and not searching* but really isn't necessary imo, cause it's searchable.

**Comment by halr9000** [comment]
Another solution I don't see mentioned above: why not just add one or more quality filters? Number of stars, likes, reviews. You would need to be smart about the design or it would get overwhelming, but if you don't want to see crap, then you could just browse things with 1+ likes, etc

**Reply by WASasquatch** [reply]
> Another solution I don't see mentioned above: why not just add one or more quality filters? Number of stars, likes, reviews. You would need to be smart about the design or it would get overwhelming, but if you don't want to see crap, then you could just browse things with 1+ likes, etc

I mentioned something like this before, though I'm not sure if it was here. Using reviews wouldn't be best, cause even huge models really don't have many reviews vs the thousands downloading them. Which is actually a shame. I wonder if that could be improved through some sort of incentive to review models. Like they get ranks/awards? 

But the immediate issue I see, is models hits/pageviews seem to greatly drop off when they're no longer featured on the top of the feed. That means hiding unverified type models to some other place, or by search, otherwise we wouldn't be protecting the integrity of good models. But that also means new users may never get established or really found. A ratio to be approved could take months of random searches by people actually looking for it. 

I think that alone would greatly impact engagement by users on civitai trying to contribute.

**Comment by kornuolis2** [comment]
- Imho if the model holds 3 and less star for more than n months, it should be archived and hidden.
- Also force minimum amount of tags.
- Allow user offer tags. Cause blockign "anime" tag is semi effective measure right now.
- Force English only in names
- Some kind of uniqueness evaluation, sending 9657th anime model to bottom of the list, and really unique model closer to the top.

**Reply by halr9000** [reply]
Agreed on minimum tags or other ways to get the metadata. Problem with archiving models without stars is that you would first need to do more to raise the % of people who write reviews. Then would be good to look at models which have downloads, but no reviews, and shine a light on those to get them either up or down voted (with stars). Then you'd have good amount of data to work with. Otherwise I think too many would be archived/invisible and site would look empty.

Ooh, how about giving users to review tags and provide missing ones?

**Reply by WASasquatch** [reply]
Yeah even super popular models here since the beginning have exponentially lowers stars then downloads. People are inherently lazy after all. Lol coming back and unloading images on a broken upload system? Lot to ask. And then sometimes more than half the bad reviews are just people having user error issues like not using right VAE, settings, or entirely not understanding what something is to begin with. Or just trying to extort prompts out of people.

**Reply by kornuolis2** [reply]
In this case system may request to evaluate the model. Something like "You have download model *Beardydwarffemales*. Please evaluate it". The more models you evaluate the more points you gain. Those points may have ranks. The more you "like", comment, reply etc the more points you have. People love shiny things and epic ranks next to their names. Like those badges on forums of all sorts.
